source/documentation/services/aws/aws-accounts.html.md.erb

```diff
@@ -23,9 +23,7 @@ The preferred method to access an AWS account is using [AWS SSO](https://moj.aws
 
 2. Confirm with the AWS Account Owner [see AWS Account List](https://docs.google.com/spreadsheets/d/1lZ2ffiN_yQEYqTYnBzTbl16iElfrp3rZ8JnznDQWc-g/edit?usp=sharing) that the requester can have access to that account.
```

Comment:
Could we remove/amend this step as well? 👀 There doesn't seem to be an owner in the spreadsheet, having the GitHub Team Maintainer as the "Account Owner" is more visible and easier to reason with 🤹

Reply:
Will raise a separate PR. It feels like the process would benefit from the addition of this info to the spreadsheet. Will update that and then amend the process. Worth noting that the AWS account Owner and GitHub Team maintainer may be completely different. This stems from the problem that we use GitHub Teams to infer RBAC permissions. The assignment of certain team is a bit arbitrary, as Teams are used for other purposes in addition to AWS access. Will have a think about it.